packages/dashboard-core-plugins/src/linker/Linker.tsx

```diff
@@ -510,6 +521,22 @@ export class Linker extends Component<LinkerProps, LinkerState> {
     }
   }
 
+  handlePanelDragging(componentId: string): void {
+    const { links } = this.props;
+    for (let i = 0; i < links.length; i += 1) {
+      const link = links[i];
+      const { start, end } = link;
+      if (start.panelId === componentId || end?.panelId === componentId) {
+        this.setState({ isDraggingPanel: true });
+        return;
+      }
+    }
```

Comment:
We probably don't even really need a check here, we could just always hide the links when dragging... but whatever, no biggie.